### DISCUSSION TITLE
No Done option in select drop down in IOS

### DISCUSSION THREAD

**Discussion by patelchetan79** [ionic-team/capacitor#3478] [Q&A]
Hi,

We have our web application on angular and ionic. We used a capacitor for ios and android compilation.  We have HTML select tag for drop-down list but in IOS when we select the value it is not auto-hide or no done button. How we can achieve this?

**Comment by jcesarmobile** [comment]
This is a side effect of Keyboard plugin hiding the form accessory bar by default.

You can show the form accessory bar with this method from Keyboard plugin

`Keyboard.setAccessoryBarVisible({isVisible: true});`

**Reply by dcyou** [reply]
https://github.com/ionic-team/capacitor/issues/2459#issuecomment-621635094
